config/invenio.conf

```diff
@@ -412,6 +412,13 @@ CFG_WEBSTYLE_REVERSE_PROXY_IPS =
 # often during the runtime.  (Note that you may modify them
 # afterwards too, though.)
 
+# CFG_WEBSEARCH_ADVANCED_SEARCH_EXACT_MATCH -- tells if the advanced
+# search should do an exact match or use the parentheses behavior (adds
+# the possibility to do boolean operation in parentheses).
+# Note that the boolean operations can always be done thanks to the form
+# in the UI.
+CFG_WEBSEARCH_ADVANCED_SEARCH_EXACT_MATCH = False
```

Comment:
If the new variable works also for the other types of queries (say regexp), it may be better to name it more generally, such as `CFG_WEBSEARCH_ADVANCED_SEARCH_PREFER_MATCHING_TYPE`.

(This is basically what the variable does: it instructs the search engine to prefer the matching type selected by the user rather than the Boolean query typed by the user.)